xf86-qubes-common/Makefile

```diff
@@ -0,0 +1,10 @@
+XORG_SERVER_CFLAGS=$(shell pkg-config --cflags xorg-server)
+
+all: libxf86-qubes-common.so
+
+libxf86-qubes-common.so: xf86-qubes-common.c include/xf86-qubes-common.h
```

Comment:
shouldn't this have fstack-protector-all, ro-relocations etc?

Reply:
rpm already define them, it's just about using CFLAGS/LDFLAGS. I'll add it here too.